benchmark.go

```diff
@@ -100,6 +104,7 @@ func runBenchmark(out output, allOpts Options, m benchmarkMethod) {
 	if opts.RPS > 0 && opts.MaxDuration > 0 {
 		// The RPS * duration in seconds may cap opts.MaxRequests.
 		rpsMax := int(float64(opts.RPS) * opts.MaxDuration.Seconds())
+		// FIXME: if we are using small time like 500ms the calculation is still 0
```

Comment:
if RPS and MaxDuration are both low enough, the product of multiplying them together will be a fraction less than `1`. e.g. `10 * time.Millisecond.Seconds() = 0.01` and the cast will truncate `int(0.01) = 0`

https://play.golang.org/p/VrpN9zrZW4